queries/models/projeto_subsidio_sppo/viagem_completa.sql

```diff
@@ -92,7 +107,14 @@ select distinct
     CURRENT_DATETIME("America/Sao_Paulo") as datetime_ultima_atualizacao
 from
     viagem_periodo v
+left join
+  shapes AS s
+using
+  (shape_id)
 where (
+    velocidade_media < {{ var("conformidade_velocidade") }} or ST_DISTANCE(s.start_pt, s.end_pt) between 1900 and 1950

```

Comment:
- Aplicar apenas apenas após `2024-11-16` (inserir data como variável)
- Não faria mais sentido algo como `velocidade_media < {{ var("conformidade_velocidade") }} and ST_DISTANCE(s.start_pt, s.end_pt) > 1900`?